packages/guilded.js/lib/managers/global/GuildBanManager.ts

```diff
@@ -37,8 +38,8 @@ export class GlobalGuildBanManager extends CacheableStructManager<string, Member
     /** Unban a user from a server. Returns existing ban if cached. */
     unban(serverId: string, userId: string, removeBanIfCached = false): Promise<MemberBan | null> {
         return this.client.rest.router.unbanMember(serverId, userId).then((data) => {
-            const existingBan = this.client.bans.cache.get(`${serverId}:${userId}`);
-            if (this.client.options.cache?.removeMemberOnLeave || removeBanIfCached) this.client.bans.cache.delete(`${serverId}:${userId}`);
+            const existingBan = this.client.bans.cache.get(buildMemberKey(serverId, userId));
```

Comment:
Can condense this line’s key and the next line’s key into the next line with a variable